src/services/api-rest/identity-resolve/identity-resolve.service.ts

```
@@ -63,10 +62,10 @@ export class IdentityResolveService {
       );
     }
 
-    const agentInfo = this.buildAgentInfo(identity, authenticationId);
+    const { agentInfo, email } = this.buildAgentInfo(identity, authenticationId);
```

ESLint

[error] Replace `identity,·authenticationId` with `⏎······identity,⏎······authenticationId⏎····`

(prettier/prettier)